src/leappto/cli.py

```diff
@@ -703,7 +708,7 @@ def print_migrate_info(text):
         if not parsed.print_port_map:
             # If we're doing an actual migration, check we have access to the
             # target, and the desired container name is available
-            check_result, claimed_names = mc.check_target()
+            check_result, claimed_names = mc.check_target_containers()
```

Comment:
Given the suggested refactoring below, this should use the new version of `mc.check_target()` so that it also ensures that the required commands are available.